src/test/java/odysseusAT/CharacterizationStepDefs.java

```diff
@@ -0,0 +1,383 @@
+    }
+
+    @When("^click to New characterization button$")
+    public void clickToNewCharacterizationButton() {
+        $(By.xpath("//*[@class='characterizations-tabbed-grid__new-entity-btn btn btn-primary btn-sm']")).click();
+        $(By.xpath("//*[@data-bind='text: title']")).shouldHave(text("New Characterization"));
+
+    }
+
+    @When("^enter Characterization name and save it$")
+    public void enterCharacterizationNameAndSaveIt() {
+        generatedString = RandomStringUtils.randomAlphanumeric(10);
+        characterizationName = "Test_" + generatedString;
+        $(By.xpath("//*[@class='input-group']/input")).clear();
+        $(By.xpath("//*[@class='input-group']/input")).setValue(characterizationName);
+        $(By.xpath("//*[@class='fa fa-save']")).click();
+    }
+
+    @When("^return to Characterization table$")
+    public void returnToCharacterizationTable() {
+        $(By.xpath("//*[@class='input-group-btn']/button[2]")).click();
+
+    }
+
+    @When("^enter created characterization name in filter$")
+    public void enterCreatedCharacterizationNameInFilter() {
+        $(By.xpath("//*[@type='search']")).waitUntil(visible, 3000);
+        $(By.xpath("//*[@type='search']")).setValue(characterizationName);
+
+    }
+
+    @Then("^can see new characterization in table$")
+    public void canSeeNewCharacterizationInTable() {
+        $(By.xpath("//*[@class=' characterizations-list__tbl-col characterizations-list__tbl-col--name ']/a")).shouldHave(text(characterizationName));
+    }
+
+    @When("^click to our characterization$")
+    public void clickToOurCharacterization() {
+        $(By.xpath("//*[@class=' characterizations-list__tbl-col characterizations-list__tbl-col--name ']/a")).click();
+
+    }
+
+    @When("^click to delete characterization button$")
+    public void clickToDeleteCharacterizationButton() {
+        $(By.xpath("//*[@class='btn btn-danger']")).waitUntil(visible, 3000);
+        $(By.xpath("//*[@class='btn btn-danger']")).click();
+
+
+    }
+
+    @When("^accept delete characterization$")
+    public void acceptDeleteCharacterization() {
+        switchTo().alert().accept();
+    }
+
+    @Then("^cant find characterization in the table$")
+    public void cantFindCharacterizationInTheTable() {
+        $(By.xpath("//*[@type='search']")).waitUntil(visible, 3000);
+        $(By.xpath("//*[@type='search']")).setValue(characterizationName);
+        $(By.xpath("//*[@class=' characterizations-list__tbl-col characterizations-list__tbl-col--name ']/a")).shouldNotHave(text(characterizationName));
+
+    }
+
+    @When("^click to Import Cohort Definition$")
+    public void clickToImportCohortDefinition() {
+        $$(byText("Import")).get(0).click();
+
+    }
+
+    @When("^choose cohort definition from the table in characterization$")
+    public void chooseCohortDefinitionFromTheTableInCharacterization() {
+        $(By.xpath("//*[@class='col-xs-6 search']/div/label/input")).setValue("test");
```

Comment:
Search for input next to `Filter:` caption

Reply:
It could work with xpath, but I don't know how I could use it with byText selector